src/main/webapp/assets/modules/post-contest/postContestTimelineDirective.js

```diff
@@ -0,0 +1,14 @@
+techlooper.directive("postContestTimeline", function ($http, utils) {
```

Comment:
'techlooper' is not defined.<br>'utils' is defined but never used.<br>'$http' is defined but never used.